### ISSUE TITLE
how to support rbac3？

### ISSUE THREAD

**Issue by wittech** [oracle/speedle#28]
did speedle support rbac3？

**Comment by caiwl** [comment]
Yes, RBAC is supported naturally by Speedle. You can assign a set of permissions to a role (or roles); define role hierarchy/SOD, etc. What RBAC feature are you looking for? @wittech
